### PR TITLE
TLS 1.2 or newer needed for this

### PR DESCRIPTION
**Pull request by eldarku**
I just solved the case where this functionality didn't work because TLS 1.0 was used.
More info here:
https://powerbi.microsoft.com/en-us/blog/deprecating-tls-1-0-and-1-1-support-in-power-bi/
So to make it work Framework 4.7 or newer needed, or, if framework 4.6 installed, change the registry according to this:
https://docs.microsoft.com/en-us/dotnet/framework/network-programming/tls#systemdefaulttlsversions

### CODE REVIEW

**Comment by PRMerger9** [comment]
@eldarku : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
Note that publishing in this repository is suspended until Tuesday, May 25. Merging will resume Tuesday. Authors can still review and sign off on PRs in the interim. Thanks.

File: docs/reporting-services/install-windows/power-bi-report-server-integration-configuration-manager.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ In addition to an active internet connection so you can browse to the [!INCLUDE[
 
 - The user performing the [!INCLUDE[sspowerbi](../../includes/sspowerbi-md.md)] integration needs to start the [!INCLUDE[ssRSnoversion](../../includes/ssrsnoversion-md.md)] Configuration Manager either with the account used to install [!INCLUDE[ssRSnoversion](../../includes/ssrsnoversion-md.md)], or the account the [!INCLUDE[ssRSnoversion](../../includes/ssrsnoversion-md.md)] service is running under
 
+- The server with Reporting Services installed need to be configured to use TLS 1.2 or newer. For more information, see [Transport Layer Security (TLS) best practices with the .NET Framework](/dotnet/framework/network-programming/tls)
```
Comment (maggiesMSFT) [inline docs/reporting-services/install-windows/power-bi-report-server-integration-configuration-manager.md:30]:
```suggestion
- The server with Reporting Services installed needs to be configured to use TLS 1.2 or newer. For more information, see [Transport Layer Security (TLS) best practices with the .NET Framework](/dotnet/framework/network-programming/tls)

```

**Review by maggiesMSFT — CHANGES_REQUESTED** [review]
please accept my edit, then merge.

**Comment by maggiesMSFT** [comment]
#sign-off   
Hi, PR Reviewers -- can you accept my change, then merge? I can't accept my change myself.

**Comment by ktoliver** [comment]
> #sign-off
> Hi, PR Reviewers -- can you accept my change, then merge? I can't accept my change myself.

Hi @maggiesMSFT - Merging is paused in this repo until Tuesday, May 25. I can do it on Tuesday; OK?

**Comment by maggiesMSFT** [comment]
Hi, @ktoliver -- OK, thanks!

**Pull request merged**